document-store/src/main/java/org/hypertrace/core/documentstore/mongo/MongoCollection.java

```diff
@@ -472,6 +469,11 @@ public Optional<Document> update(
       final org.hypertrace.core.documentstore.query.Query query,
       final java.util.Collection<SubDocumentUpdate> updates)
       throws IOException {
+
+    if (updates.isEmpty()) {
```

Comment:
I hope it doesn't break any existing updates from upstream services

Reply:
That's correct. No consumers as of now.